packages/fetch/src/fetch.ts

```diff
@@ -1,4 +1,4 @@
-import { AbortSignal, createAbortSignal, withLabels, Operation, Resource } from '@effection/core';
+import { AbortSignal, createAbortSignal, withLabels, OperationObject, Resource } from '@effection/core';
```

Comment:
`OperationObject` is a private type, so none of our public interfaces should return it explicitly. Let's revert this back to `Operation`

Reply:
@cowboyd Looks like you saw some of old commit. The actual version of PR doesn't export `OperationObject` from `core` nor uses it in `fetch` and `exec`.